pkg/plugins/client.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 const (
-	defaultTimeOut = 30
+	defaultTimeOut     = 30
+	defaultHTTPTimeOut = 3
```

Comment:
3s is low, especially given that we are more lenient with non-blocking http errors (max of 16s). 

go-connections uses 32s for default socket timeouts. https://github.com/docker/docker/pull/8035 Lets do the same.